packages/@aws-cdk/aws-iotevents/test/integ.detector-model.ts

```diff
@@ -22,20 +41,20 @@ class TestStack extends cdk.Stack {
             iotevents.Expression.fromString('31.5'),
           ),
         ),
+        actions: [setTemperatureAction],
       }],
     });
     const offlineState = new iotevents.State({
       stateName: 'offline',
     });
 
-    // 1st => 2nd
     onlineState.transitionTo(offlineState, {
       when: iotevents.Expression.eq(
         iotevents.Expression.inputAttribute(input, 'payload.temperature'),
         iotevents.Expression.fromString('12'),
       ),
+      executing: [setTemperatureAction],
     });
-    // 2st => 1st
     offlineState.transitionTo(onlineState, {
       when: iotevents.Expression.eq(
         iotevents.Expression.inputAttribute(input, 'payload.temperature'),
```

Comment:
Can we just leave this test unchanged? We already have an integ test in the `@aws-cdk/aws-iotevents-actions` module, I don't think we need this too.